saleor/events/__init__.py

```diff
@@ -0,0 +1,148 @@
+from django.utils.translation import pgettext_lazy
```

Comment:
Do we really need a special module only for keeping up to date history of changes in order? I understand that in further steps we could have CustomerEvent etc, but even though I prefer to keep this logic inside a given module. Personally, it is more intuitive.
1) For now, module events operates only on Order's events.  So it can be easily moved to Order module.
2) If you really need base class: Event Manager, you can put it inside the core module.
3) If you put OrderEvents inside Order module, there is no need to modify so many old migrations, which personally I don't like